examples/cdn/index.html

```diff
@@ -55,21 +55,10 @@
   );
 }
 
-const useStyles = makeStyles(theme => ({
-  root: {
-    margin: theme.spacing(6, 0, 3),
-  },
-  lightBulb: {
-    verticalAlign: 'middle',
-    marginRight: theme.spacing(1),
-  },
-}));
-
 function ProTip() {
-  const classes = useStyles();
   return (
-    <Typography className={classes.root} color="textSecondary">
-      <LightBulbIcon className={classes.lightBulb} />
+    <Typography sx={{ mt: 6, mb: 3 }} color="textSecondary">
+      <LightBulbIcon sx={{ mr: 1, verticalAlign: 'top' }} />
```

Comment:
```suggestion
      <LightBulbIcon sx={{ mr: 1, verticalAlign: 'middle' }} />
```
@oliviertassinari ?

Reply:
Yeah, I also took the opportunity to improve the display 👼

middle:
<img width="506" alt="Capture d’écran 2021-01-28 à 17 00 30" src="https://user-images.githubusercontent.com/3165635/106164547-60a8fa00-618a-11eb-9a5d-73c9d425862a.png">

top:
<img width="500" alt="Capture d’écran 2021-01-28 à 17 00 38" src="https://user-images.githubusercontent.com/3165635/106164552-61419080-618a-11eb-8260-ed2157210542.png">

Reply:
Ah nice, wasn't sure if it is intended. Ok, resolving